install/merger/types/configmap_override.go

```diff
@@ -0,0 +1,31 @@
+package types
+
+import (
+	"context"
+	. "github.com/kyma-incubator/hydroform/install/util"
```

Comment:
I think using dot in file import is not recommended in this case. Please see: https://github.com/golang/go/wiki/CodeReviewComments#import-dot